beacon_node/network/src/beacon_processor/worker/rpc_methods.rs

```diff
@@ -671,17 +798,45 @@ impl<T: BeaconChainTypes> Worker<T> {
                     error!(
                         self.log,
                         "No blobs or block in the store for block root";
+                        "request" => %req,
+                        "peer" => %peer_id,
+                        "block_root" => ?root
+                    );
+                    break;
```

Comment:
id this is an error (from the log) don't we want to send some kind of error to the peer?

Reply:
this is also the same as how we handle blocks by range, i'll send you a prior conversation about this